<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tcp_Server</class>
 <widget class="QWidget" name="Tcp_Server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>209</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>IP地址：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ip_edit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>210</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>端口号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="port_edit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>179</y>
     <width>121</width>
     <height>256</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>机器人位置：</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>X：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="x_edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Y：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="y_edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Z：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="z_edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>RX：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="rx_edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>RY：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ry_edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>RZ：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="rz_edit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="send_btn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>440</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="register_tb">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>40</y>
     <width>201</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="start_server_btn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>开启服务器</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>154</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>机器人：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="robot_model_box"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>179</y>
     <width>151</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>焊缝信息：</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>延时：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="delay_edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>搜索状态：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="search_state_edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>焊缝Y：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="weld_y_edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>焊缝Z：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="weld_z_edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>焊缝高度：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="weld_h_edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_15">
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>焊缝宽度：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="weld_w_edit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>124</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QRadioButton" name="modbus_tcp_rbtn">
      <property name="text">
       <string>modbus_tcp</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="kawasaki_json_rbtn">
      <property name="text">
       <string>kawasaki_json</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="write_pos_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>439</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>写入位置信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="write_weld_btn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>439</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>写入焊缝信息</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="record_tb">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>241</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>430</y>
     <width>201</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_16">
    <item>
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>地址：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="reg_addr_edit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>值：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="reg_value_edit"/>
    </item>
    <item>
     <widget class="QPushButton" name="write_reg_btn">
      <property name="text">
       <string>写入</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>寄存器：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
